how-to/release/mozillavpn.rst

```diff
@@ -0,0 +1,211 @@
+Mozilla VPN Release Process
+===========================
+
+`Mozilla VPN`_ is shipped in two parts, the client and addons.
+
+The client is shipped on Windows, Mac and Linux, as well as Android and iOS.
+Ensure you are in the ``#mozilla-vpn-release`` channel in Slack.
+
+.. _vpn relpro:
+
+Find the Release Promotion Action
+---------------------------------
+
+Many of the steps below use a `release promotion action`_, these all share the
+same initial steps:
+
+1. Navigate to the Decision task of the push in Taskcluster. Ensure it is the
+   one for the ``github-push`` event, it will be called ``Decision (push)``.
+2. Navigate to the task group.
+3. From the three dot menu in the bottom right, select ``Release Promotion``.
+
+Windows and Mac Clients
+-----------------------
+
+Promote a Build
+~~~~~~~~~~~~~~~
+
+You'll receive a ping in the ``#mozilla-vpn-release`` channel in Slack asking to
+"promote a build" or "push a build to candidates". The request should include a
+link to the commit they would like promoted.
+
+1. :ref:`vpn relpro`.
+2. Change the ``release_promotion_flavor`` to ``promote-client``.
+3. Trigger the action. This will run beetmover tasks that copy the signed
+   builds over to the `candidates directory`_ on archive.mozilla.org. The
+   builds will be under the path with the following pattern:
+   ``pub/vpn/candidates/<version>-candidates/build<buildId>/``
+
+4. :ref:`vpn balrog`, making sure to use the `staging instance`_.
+5. Reply in-thread that the candidate builds are ready for testing.
+
+Ship a Build
+~~~~~~~~~~~~
+
+You'll receive a ping in the ``#mozilla-vpn-release`` channel in Slack asking to
+ship or release a build. The request should include a link to the commit they
+would like promoted.
+
+.. note::
```

Comment:
We could disable these signing tasks on-push for mac/windows/addons? When we run the promotion they would get back-filled

Reply:
Yeah, that's the plan. I have a WIP PR for this and will update the note after it lands.